installing/installing_sno/install-sno-installing-sno.adoc

```diff
@@ -35,6 +35,12 @@ To install {product-title} on a single node, first generate the installation ISO
 
 include::modules/install-sno-generating-the-install-iso-manually.adoc[leveloffset=+2]
 
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../../post_installation_configuration/enabling-cluster-capabilities.adoc[Enabling cluster capabilities]
+* xref:../../installing/cluster-capabilities.html#explanation_of_capabilities_cluster-capabilities[Optional cluster capabilities in OpenShift Container Platform Branch Build]
+
```

Comment:
Is it possible to add two doc links with more explanation as what you did in line 116 and line 117 in installing-bare-metal.adoc?

Reply:
@jinyunma 
Thanks for the suggestion - yes I've done that